test_util/cluster.py

```diff
@@ -454,3 +457,22 @@ def run_integration_tests(cluster, setup=True, **kwargs):
             public_agent_list=[h.private_ip for h in cluster.public_agents],
             provider=cluster.provider,
             **kwargs)
+
+
+def get_apps_tasks_details(cluster, install_app=False):
```

Comment:
I am very confused by the API of this method. Why is this not a member of cluster if it relies so heavily on cluster for its inputs?  This just calls cluster.marathon.get_app_tasks_details() anyway, so the only difference is that it sometimes deploys an app?  I dont even see this being used....